utils/test_vectors/src/test-vectors.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "cipher_suite": 1,
+    "cipher_suite": "AesCm128HmacSha256_8",
```

Comment:
I would not modify the `json` file per se as this is the direct output from the  generator script in the `sframe` repo ([they even checked it in ](https://github.com/sframe-wg/sframe/blob/main/test-vectors.json)). If something changes there (e.g through [https://github.com/sframe-wg/sframe/pull/106](this PR) :P) we would have to do so again. Maybe add a custom `serde` deserialze function where we map those numbers to the according strings?

Reply:
I found some time and took the liberty to change that :)